scss/components/_navbar.scss

```diff
@@ -6,7 +6,29 @@
  * of all or any part of the contents of this file is strictly prohibited.
  */
 
-.navbar {
+body {
+
+  > .navbar {
```

Comment:
you can not assume that a navbar will always be a direct descendant of the body thus raising specificity this way might never create the desired effect

Reply:
The point of this is to target only navbars that are the direct descendent of the body. These styles are for creating the top horizontal navbar that is intended to contain the global breadcrumbs and such. It must be a direct descendent of the body (and be a sibling of the .navbar-primary) for the Fluid navbar collapse/expand to work properly. This is all part of the recommended/assumed layout that Fluid apps should use, and our demo page is providing an example of it.

Reply:
What if you can't get it as a direct descendant of the body? There are some frameworks that force a wrapper around the whole application

Reply:
OK, I'll have to add another custom class for the header and footer then.